### DISCUSSION TITLE
Render as markdown?

### DISCUSSION THREAD

**Discussion by dirkarnez** [yuin/goldmark#283] [Q&A]
I would like to use this library for editing a markdown file programmatically, this library allows me to do the editing, but i cannot find a function to render the document (`ast.Node`) to markdown, instead of HTML.

**Comment by wdscxsj** [comment]
Same idea [here](https://github.com/yuin/goldmark/discussions/265).

**Comment by iwahbe** [comment]
You can use a different renderer. I have been using https://github.com/teekennedy/goldmark-markdown.

**Reply by marco-m** [reply]
Thanks @iwahbe for the link!
